src/cli/cli_commissioner.cpp

```diff
@@ -345,17 +345,22 @@ void Commissioner::HandleStateChanged(otCommissionerState aState, void *aContext
 void Commissioner::HandleStateChanged(otCommissionerState aState)
 {
     mInterpreter.mServer->OutputFormat("Commissioner: ");
+    PrintState(aState);
+    mInterpreter.mServer->OutputFormat("\r\n");
+}
 
+void Commissioner::PrintState(otCommissionerState aState)
```

Comment:
nit: Looks like having a function coverts the `State` to its string representation can make the printing code simpler :)

Reply:
Are you proposing to make this a static function rather than a private method?

Reply:
oh, maybe a function like this:
```C++
const char *Commissioner::StateToString(otCommissionerState aState)
{
    switch (aStahe)
    {
    case kDisabled: return "Disabled";
    ...
    }
}
```

and we can simplify the printing code:
```c++
    mInterpreter.mServer->OutputFormat("Commissioner: %s\r\n", StateToString(aState));
```